activiti-cloud-events-adapter/values.yaml

```diff
@@ -0,0 +1,50 @@
+# Default values for Maven projects.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+
+global:
+  rabbitmq:
+    host: ""
+    username: guest
+    password: guest
+    destination: engineEvents
+    group: eventsAdapter
+    contentType: application/json
+  activemq:
+    host: ""
+    username: admin
+    password: admin
+    topicName: alfresco.events.source
+
+javaOpts:
+  xmx: 512m
+  xms: 512m
+  other: -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true -XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90
+
+image:
+  repository: activiti-cloud-events-adapter
+  tag: latest
+  pullPolicy: IfNotPresent
+service:
+  name: event-adapter
+  type: NodePort
```

Comment:
@rgauss @cristina-sirbu I don't like this.. because this is Minikube specific. By default it should be ClusterIP .. meaning no value inside the values.yaml file.. Does this make sense?

Reply:
Removed the service type specification (though the other charts do seem to set it).